### PR TITLE
Ping Annotations

### PR DESCRIPTION
**Pull request by wlach**

### CODE REVIEW

File: scripts/create-api.py
Comment (acmiyaguchi) [inline scripts/create-api.py:11]: It might be worthwhile to create a schema to validate the API:

```yaml
type: object
description: application
additionalProperties:
  type: object
  description: metric/ping
  additionalProperties:
    type: object
    properties:
      content:
        type: string
        description: markdown contents of annotation
      component:
        type: string
        $comment: could be an enumerated type
      features:
        type: string
    required:
      - content
```

**Review by acmiyaguchi — APPROVED** [review]
This looks reasonable to me.

File: scripts/create-api.py
Comment (wlach) [reply scripts/create-api.py:11]: Great idea! Filed https://github.com/mozilla/glean-annotations/issues/9 for this

**Comment by wlach** [comment]
Waiting for review on mozilla/glean-dictionary#493 before merging this: they mutually depend on each other.

**Pull request merged**
